### DISCUSSION TITLE
Let us break more things by scripting!

### DISCUSSION THREAD

**Discussion by Fierelier** [multitheftauto/mtasa-blue#2722] [Ideas] [answered]
### Is your feature request related to a problem? Please describe.

I often wanna do stuff client-side, to save on bandwidth. What I want to do could cause desync, but it would still be useful. Sometimes functionality, like setting animation groups on vehicles, is removed because some people cannot use it. I think that sucks and just limits the possibilities for people who know what they're doing.

### Describe the solution you'd like

Just let us do more stuff client side, like changing vehicle handling, even if it doesn't make much sense to do so. There is already so much stuff that can cause desyncs, like setting element models for example. Just lift the client side limitations and let us go all the way. Also, do not remove functionality that **can** break the game.

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not about a cheat or security vulnerability.

**Comment by dmi7ry** [comment]
Yep, it can even cause crashes in some cases (see `animGroup` of `setVehicleHandling`), although the `animGroup` may be useful in some cases (actually I used it).

**Comment by Fernando-A-Rocha** [comment]
post it as an Issue of type suggestion

**Comment by qaisjp** [comment, marked as answer]
Yes, we're ok with adding features that may behave strangely to the client Lua API. There are many use cases, e.g. Sphene.
